languages/swift/exercises/concept/structs-and-classes/.docs/introduction.md

```diff
@@ -0,0 +1,158 @@
+Structs and classes are two of the primary building blocks of Swift programming. They are both means of grouping together related data and functions into self-contained units of functionality. All of the basic types that you have been working with up to this point, such as `Int`, `Double`, `Bool`, and `Character` as well as collection types like `Array`, `String`, and `Dictionary` are actually structs. And when you define a struct or class, you are defining a new type to be used within Swift, just as you used those types.
```

Comment:
Does the student need to know that `Int`, `Double`, etc. are all actually structs in order to be able to solve the exercise? If not, you could remove it here and keep it in the `after.md`, which is perfect for this type of information.

Reply:
Rewritten in the introduction, left as is in the after.